enterprise/templates/enterprise/admin/manage_learners.html

```diff
@@ -100,10 +100,10 @@
 </div>
 
 <div class="forms-panel">
-  <h1>{% trans "Link learner" %}</h1>
-  <form action="" method="post">
+  <h1>{% trans "Link learners" %}</h1>
+  <form action="" method="post" enctype="multipart/form-data">
     {% csrf_token %}
-    {{ manage_learners_form }}
+    {{ manage_learners_form.as_p }}
```

Comment:
optional: we can add a line comment here: `as_p will render them wrapped in <p> tags`